src/views/forms/TriggersForm/TriggersForm.tsx

```diff
@@ -0,0 +1,119 @@
+
+import { type SubaccountOrder } from '@/constants/abacus';
+import { ButtonAction } from '@/constants/buttons';
+import { STRING_KEYS } from '@/constants/localization';
+
+import { useStringGetter } from '@/hooks';
+
+import { layoutMixins } from '@/styles/layoutMixins';
+
+import { Button } from '@/components/Button';
+import { Output, OutputType } from '@/components/Output';
+
+import { getPositionDetails } from '@/state/accountSelectors';
+
+import { AdvancedTriggersOptions } from './AdvancedTriggersOptions';
+import { TriggerOrderInputs } from './TriggerOrderInputs';
+
+type ElementProps = {
+  marketId: string;
+  stopLossOrders: SubaccountOrder[];
+  takeProfitOrders: SubaccountOrder[];
+  onViewOrdersClick: () => void;
+};
+
+export const TriggersForm = ({ marketId, stopLossOrders, takeProfitOrders, onViewOrdersClick }: ElementProps) => {
+  const stringGetter = useStringGetter();
+
+  const { asset, entryPrice, stepSizeDecimals, tickSizeDecimals, oraclePrice } =
+    useSelector(getPositionDetails(marketId)) || {};
+  const symbol = asset?.id ?? '';
+
+  const isDisabled = false; // TODO: CT-625 Update based on whether values are populated based on abacus
+
+  const priceInfo = (
+    <Styled.PriceBox>
+      <Styled.PriceRow>
+        <Styled.PriceLabel>{stringGetter({ key: STRING_KEYS.AVG_ENTRY_PRICE })}</Styled.PriceLabel>
+        <Styled.Price type={OutputType.Fiat} value={entryPrice?.current} />
+      </Styled.PriceRow>
+      <Styled.PriceRow>
+        <Styled.PriceLabel>{stringGetter({ key: STRING_KEYS.ORACLE_PRICE })}</Styled.PriceLabel>
+        <Styled.Price type={OutputType.Fiat} value={oraclePrice} />
+      </Styled.PriceRow>
+    </Styled.PriceBox>
+  );
+
+  return (
+    <Styled.Form>
+      {priceInfo}
+      {
+        <TriggerOrderInputs
+          symbol={symbol}
+          tooltipId={'take-profit'}
+          stringKeys={{
+            header: STRING_KEYS.TAKE_PROFIT,
+            price: STRING_KEYS.TP_PRICE,
+            output: STRING_KEYS.GAIN,
+          }}
+          orders={takeProfitOrders}
+          onViewOrdersClick={onViewOrdersClick}
+        ></TriggerOrderInputs>
+      }
+      {
+        <TriggerOrderInputs
+          symbol={symbol}
+          tooltipId={'stop-loss'}
+          stringKeys={{
+            header: STRING_KEYS.STOP_LOSS,
+            price: STRING_KEYS.SL_PRICE,
+            output: STRING_KEYS.LOSS,
+          }}
+          orders={stopLossOrders}
+          tickSizeDecimals={tickSizeDecimals}
+          onViewOrdersClick={onViewOrdersClick}
+        ></TriggerOrderInputs>
+      }
+      {
+        <AdvancedTriggersOptions
+          symbol={symbol}
+          stepSizeDecimals={stepSizeDecimals}
+          tickSizeDecimals={tickSizeDecimals}
+        ></AdvancedTriggersOptions>
+      }
+      <Button action={ButtonAction.Primary} state={{ isDisabled }}>
+        {isDisabled
+          ? stringGetter({ key: STRING_KEYS.ENTER_TRIGGERS })
```

Comment:
I feel like there is  an intermediate step before being disabled that should control the CTA.
Like having a var `hasInputtedTriggers` that is part of the `isDisabled` check.

Reply:
ah yeah - I was going to update that logic as a part of the TODO [here](https://github.com/dydxprotocol/v4-web/pull/376/files#diff-d711aaf1fda8e97dbf752a4b79c17ff1156d55a1df2174db83f0ab8e87a73ab2R34), but I can make that more clear in my variable naming for rn